Regression/Checksum/benchmarks_json/test_rz_laser_injection_from_RZ_lasy_file.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
@dpgrote Could you explain why this needed to be changed?

Reply:
Because of the change in the boundary condition. This has been moved to PR #5886